release/changelogparser.py

```diff
@@ -56,11 +66,16 @@ def parse_line(line, line_num):
 
 version_line_re = re.compile(r'^(v[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+)?) - ([0-9]{4}-[0-9]{2}-[0-9]{2})( \(aborted release/not recommended\))?$')
 
+def is_buildkit_change_additional_info_required(version):
+    return version != "Unreleased" and packaging.version.parse(version) >= packaging.version.parse('v0.7.19')
```

Comment:
```suggestion
    return version != "Unreleased" and packaging.version.parse(version) > packaging.version.parse('v0.7.19')
```
?

Reply:
I edited the CHANGELOG for v0.7.19 to include this section already. I think it's fine as-is -- the important thing is it'll remind us to do this next time.